src/struct/commands/arguments/TypeResolver.js

```diff
@@ -122,6 +122,58 @@ class TypeResolver {
                 return color;
             },
 
+            [ArgumentTypes.TIME]: (message, phrase) => {
+                if (!phrase) return null;
+
+                const TIME_UNITS = {
+                    years: {
+                        label: '(?:y|years?|year?)',
```

Comment:
This will allow matching on `y`, `years`, `year`, and `yea`, the latter of which I assume isn't intended.

Reply:
![image](https://user-images.githubusercontent.com/62367547/95151356-ab0a0380-07b4-11eb-9959-7e61d917022f.png)
 hmmm i dont think so

Reply:
eh wait it work...